targets/TARGET_Realtek/mbed_rtx.h

```diff
@@ -33,8 +33,7 @@
     #define INITIAL_SP          (__StackTop)
 #endif
 
-
-#if defined(__CC_ARM) || defined(__GNUC__)
```

Comment:
Was this redundant ? `ISR_STACK_SIZE` can be set for ARMCC or ARM GCC (see mbed boot code file)

Reply:
@0xc0170 ISR_STACK_SIZE is always set in the case of CC_ARM and ARMCC above.
\

Reply:
I see now. Then this should be moved to the above (ARMCC provides gnu flag that defines also GNUC macro that we use) or be completely removed? I would say to be removed, the default stack size is set. Thus not needed here ?